src/contracts/misc/interfaces/IGhoAaveSteward.sol

```diff
@@ -0,0 +1,140 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+/**
+ * @title IGhoAaveSteward
+ * @author Aave Labs
+ * @notice Defines the basic interface of the GhoAaveSteward
+ */
+interface IGhoAaveSteward {
+  /**
+   * @notice Emitted when the borrow rate configuration has been updated
+   * @param optimalUsageRatioMaxChange The new allowed max percentage change for optimal usage ratio
+   * @param baseVariableBorrowRateMaxChange The new allowed max percentage change for base variable borrow rate
+   * @param variableRateSlope1MaxChange The new allowed max percentage change for variable rate slope 1
+   * @param variableRateSlope2MaxChange The new allowed max percentage change for variable rate slope 2
+   */
+  event BorrowRateConfigSet(
```

Comment:
this event is not needed/consistent. We are not emitting event in any of the other functions

Reply:
Done